bq-connector/docai_bq_connector/exception/TableNotFoundError.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2022 Google LLC
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+class TableNotFoundError(Exception):
```

Comment:
Same comment as above:

Is this exception really needed? won't the Google Cloud APIs throw a `NOT FOUND` or `INVALID ARGUMENT` exception for this?

Reply:
The intent behind this was to provide a clearer exception to what was happening. I can eliminate, but it definitely helps clarify the issue.

Reply:
I suppose that makes sense, go ahead and keep this.